src/runtime_src/core/include/xrt/xrt_device.h

```diff
@@ -103,6 +107,8 @@ enum class device : unsigned int {
   platform,
   pcie_info,
   host, 
+  aie, 
+  aie_shim, 
```

Comment:
I want to make sure that we really want to expose this info to end-user and that @uday610 is aware of this.
I am fine adding these, but is there an ask for host applications to be able to access these info datas?

If these are needed only from xbutil and profiling, then internal access can be provided.  But if these are indeed needed by end-user applications, then keep as is.

Reply:
I believe, The request is to add aie and aie_shim support for **get_info** API.